Path is not getting expanded properly.

jail-task-driver is seeing:
```"{Path:/ local/.. ```
but the config is set:
```Path    = "${NOMAD_TASK_DIR}/.."```

I think if we set:
https://github.com/cneira/jail-task-driver/blob/2eec69fd464544c062f82352bc1a6d3aea3d15e2/driver/jail.go#L88
for the default to be TaskConfig.AllocDir if Path is unset, that would fix my issue.

I tried that:
```diff
diff --git a/driver/jail.go b/driver/jail.go
index 1d4480a..37ebf88 100644
--- a/driver/jail.go
+++ b/driver/jail.go
@@ -82,7 +82,11 @@ func (d *Driver) initializeContainer(cfg *drivers.TaskConfig, taskConfig TaskCon

        jailparams["name"] = fmt.Sprintf("%s-%s", cfg.Name, cfg.AllocID)
        jailparams["host.hostname"] = fmt.Sprintf("%s-%s", cfg.Name, cfg.AllocID)
-       jailparams["path"] = taskConfig.Path
+       if len(taskConfig.Path) > 0 {
+               jailparams["path"] = taskConfig.Path
+       } else {
+               jailparams["path"] = taskConfig.AllocDir
+       }

        if len(taskConfig.Jid) > 1 {
                jailparams["jid"] = taskConfig.Jid
```
but I get a 
driver/jail.go:88:34: taskConfig.AllocDir undefined (type TaskConfig has no field or method AllocDir)
but the docs seem to say it exists:
https://godoc.org/github.com/hashicorp/nomad/plugins/drivers#TaskConfig

I don't know enough go to understand why AllocDir doesn't seem to exist..

--- a/driver/jail.go
+++ b/driver/jail.go
@@ -14,6 +14,7 @@ import (
 	"github.com/hashicorp/nomad/plugins/drivers"
 	"os/exec"
 	"strings"
+"path/filepath"
 )
 
 func simple_uuid() (string, error) {
@@ -102,7 +103,7 @@ func (d *Driver) initializeContainer(cfg *drivers.TaskConfig, taskConfig TaskCon
 	if len(taskConfig.Path) > 0 {
 		jailparams["path"] = taskConfig.Path
 	} else {
-		jailparams["path"] = cfg.AllocDir
+		jailparams["path"] = filepath.Join(cfg.AllocDir, cfg.Name)
 	}
 
 	if len(taskConfig.Jid) > 1 {

